### PR TITLE
+1 team  member

### PR DESCRIPTION
**Pull request by following5**
### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code

### CODE REVIEW

File: htdocs/templates2/ocstyle/articles/EN/team.tpl [hunk 1]
```diff
@@ -67,6 +67,12 @@
                 </td>
             </tr>
             <tr>
+                <td><img src="{$opt.page.teampic_url}dummy.jpg"/></td>
+                <td><a href="{$opt.page.teammember_url}viewprofile.php?userid=150360">following</a><br/>
+                    data maintenance, software development
```
Comment (teiling88) [inline htdocs/templates2/ocstyle/articles/EN/team.tpl:72]: please only development like the other entries.

Comment (MacGyver-NRW) [reply htdocs/templates2/ocstyle/articles/EN/team.tpl:72]: Would be nice to check Slini as well ;)
It's same for him. @teiling88

Comment (teiling88) [reply htdocs/templates2/ocstyle/articles/EN/team.tpl:72]: oh my fault.

Comment (teiling88) [reply htdocs/templates2/ocstyle/articles/EN/team.tpl:72]: merged and deployed to live

**Pull request merged**